libs/wingc/src/type_check.rs

```diff
@@ -266,7 +267,7 @@ pub struct Namespace {
 	pub name: String,
 
 	#[derivative(Debug = "ignore")]
-	pub env: SymbolEnv,
+	pub envs: Vec<SymbolEnvRef>,
```

Comment:
@yoav-steinberg FYI ☝️  - this is how I figured how to add support for multi-file imports, curious if you see any dangers down this implementation path...

Reply:
My initial concern is that this has the potential to break the name uniqueness we're trying to have at a namespace level. In theory now a namespace might have two symbols with the same name each in a different env.
I'll dig into the implementation to see if I'm missing something here.